docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md

```diff
@@ -54,7 +54,7 @@ Note: Please create a space and define the schema in Nebula Graph first, then us
 
 #### Vertices
 
-A vertex data file consists of multiple rows, with each line in the file representing a point and its properties. In general, the first column is the ID of the vertex. This ID column is specified in the mapping file. Other columns are the properties of the vertex. Consider the following example in JSON format.
+A vertex data file consists of multiple rows. Each line in the file represents a vertex and its properties. In general, the first column is the ID of the vertex. This ID column is specified in the mapping file. Other columns are the properties of the vertex. Consider the following example in JSON format.
```

Comment:
represents one vertex and its properties. 

"Other columns" ==> "The other columns"